Auth/FirebaseAuthUI/FUIAuth.m

```diff
@@ -197,9 +197,9 @@ - (void)signInWithProviderUI:(id<FUIAuthProvider>)providerUI
                                   credential:credential
                               resultCallback:result];
     } else {
-      [self.auth signInWithCredential:credential
-                           completion:^(FIRAuthDataResult *_Nullable authResult,
-                                        NSError *_Nullable error) {
+      [self.auth signInAndRetrieveDataWithCredential:credential
```

Comment:
signInAndRetrieveDataWithCredential: is deprecated, see my reply in b/

Reply:
Will clean this up--I merged early so other PRs could have passing tests.

Reply:
https://github.com/firebase/FirebaseUI-iOS/pull/690